CHANGELOG.md

```diff
@@ -1,5 +1,48 @@
+* AKS will be upgrading the KEDA addon to more recent KEDA versions. The AKS team has added KEDA 2.15 on AKS clusters with K8s versions >=1.31, KEDA 2.14 for Kubernetes v1.30. KEDA 2.15 and KEDA 2.14 will introduce multiple breaking changes which are listed below:
+  * **KEDA 2.15** for Kubernetes >=1.31: The removal of [Pod Identity support](https://github.com/kedacore/keda/issues/5035). If you use pod identity, we recommend you move over to [workload identity for your authentication](https://learn.microsoft.com/azure/aks/keda-workload-identity). 
+  * **KEDA 2.14** for Kubernetes = 1.30: The removal of [Azure Data Explorer 'metadata.clientSecret' as it was not safe for managing secrets](https://github.com/kedacore/keda/issues/4514).
+  * **KEDA 2.14** for Kubernetes = 1.30: Removal of the [deprecated metricName from trigger metadata section](https://github.com/kedacore/keda/issues/4240). The two impacted Azure Scalers are Azure Blob Scaler and Azure Log Analytics Scaler. If you are using `metricName` today, please move `metricName` outside of trigger metadata section  to`trigger.name` in the trigger section to optionally name your trigger. To view an example of what this would look like, please view the open [GitHub issue](https://github.com/Azure/AKS/issues/4471).
+
+### Release Notes
+
+* Features:
+  * AKS patch versions 1.28.13, 1.29.8, 1.30.4 are now available. Refer to [version support policy](https://learn.microsoft.com/azure/aks/supported-kubernetes-versions?tabs=azure-cli#kubernetes-version-support-policy) and [upgrading a cluster](https://learn.microsoft.com/azure/aks/upgrade-aks-cluster?tabs=azure-cli) for more information.
+
+* Bug fixes:
+  * Updated the ACNS Image tag for fixing the [bug](https://github.com/Azure/AKS/issues/4525) that causes cilium pods to creash in ACNS enabled AKS clusters
+  * Bug fix to address the issue where the OSDiskSize validator throws an error if the existing agent pool does not have a default value set
+  
+* Behavior change:
+  * Abandoned cluster will be deallocated with status `Failed(Deallocated)` instead of `Succeeded (Stopped)`.
+  * PDB drain errors will now include additional PDB debug message and appropriate original error instead of generic "API call to Kubernetes API Server failed" error message.
+  * Updated [Azure NPM version to v1.5.36](https://github.com/Azure/azure-container-networking/releases/tag/v1.5.36) to address race condition in Azure NPM Linux which can occur when editing/deleting a NetworkPolicy with "enough" rules. The race can result in unexpected connectivity for traffic to/from Pods on the impacted Node. NPM will now auto-restart to mitigate the issue ~15 seconds after if it enters a broken state caused by the race.
+  * In k8s 1.31, we will default to parallel image pulling, by setting the kubelet flag --serialize-image-pulls = false(default value is true)
```

Comment:
This maybe should be an announcement as its a larger default behavior change. And any documentation @stl327?

Reply:
Also should clarify we are changing the default value, not just setting it.

Reply:
Do we have docs for this @allyford  - few customers are interested in this.

Reply:
@allyford - confirmed with Spencer. Removing this as 1.31 didn't go this week. Please add it in the next release notes